package-lock.json

```diff
@@ -16,7 +16,7 @@
       "integrity": "sha1-r9+UiPsezvyDSPb7IvRk4ypYs2s=",
       "dev": true,
       "requires": {
-        "acorn": "3.3.0"
```

Comment:
Huh interesting. `save-exact = true` made no difference, it's because I installed npm 6 last week when switching a different machine. Apparently this is a change in v6 that doesn't affect the actual installed version of the dependency. See here for more details (h/t @kencheeto): https://github.com/npm/npm/issues/20434. 

I'm just downgrading to v5 since Ken said we'd have to update npm on our servers as well if we decided to go to v6 (and it's probably not stable yet anyway).